users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -139,8 +155,11 @@ def apply_codecs(audio, sample_rate, random_state, codecs):
                 if random_state.random() < prob:
                     mu = random_state.random() * (max_value - min_value) + min_value
                 else:
-                    # standard value for mu-law encoding
-                    mu = 255
+                    if codec.get("default") is None:
+                       continue
+                    else:
+                        # standard value for mu-law encoding
+                        mu = codec.get("default")
```

Comment:
Slightly nicer imho.
```suggestion
                    # standard value for mu-law encoding
                    mu = codec.get("default")
                    if mu is None:
                       continue
```